src/sindri/config/website.py

```diff
@@ -411,16 +410,16 @@
             },
         "fast_update": False,
         },
-    "chargestate": {
+    "ledstate": {
         "plot_type": "numeric",
         "plot_data": {
             "delta_period": "1H",
             "threshold_period": "24H",
             "threshold_type": "max",
-            "variable": "charge_state",
+            "variable": "led_state",
             },
         "plot_metadata": {
-            "plot_title": "Charge State",
+            "plot_title": "Battery LED State",
```

Comment:
```suggestion
            "plot_title": "LED State",
```

Space is very tight here and the LED shows more than just the battery status (AFAIK) so I suggest shortening this